csrc/ops/arith.h

```diff
@@ -770,4 +771,13 @@ TORCH_CUDA_CU_API TensorView* fusedMultiplySum(
     const std::vector<int>& axes,
     Val* init = nullptr);
 
+// Create a tensor view from the given value. The given value can be a single
+// scalar, an array of scalars, or a nested array of scalars.
+TensorView* tensor(Val* val);
```

Comment:
So, if it's a scalar, it will create a 0-dim tensor and the scalar value is used to initialize the 0-dim tensor. If it's an array, the value is used to determine the extent of each dimension. Is this correct?

Reply:
Yes, for example if you have `val` with data type `Array<Array<Float, 5>, 3>`, then you will have a float tensor with shape `(5, 3)`.